File: src/main/java/com/travel_stories/ServerTimeLineEntry.java
package com.travel_stories;

import java.io.Serializable;
import java.math.BigInteger;
import java.util.ArrayList;
import java.util.GregorianCalendar;
public class ServerTimeLineEntry implements Serializable{

    public final ArrayList<String> photos;
    public final String locationName;
    public final BigInteger location;
    public final GregorianCalendar start;
    public final GregorianCalendar end;


    public ServerTimeLineEntry(ArrayList<String> photos, String locationname, BigInteger location, GregorianCalendar start, GregorianCalendar end){
        this.photos = photos;
        this.locationName = locationname;    	
    	this.location = location;
        this.start = start;
        this.end  = end;
    }
    
}

File: src/main/scala/com/travel_stories/ServerTimeLineEntry.scala
package com.travel_stories;

import java.io.Serializable;
import java.math.BigInteger;
import java.util.ArrayList;
import java.util.GregorianCalendar;

class ServerTimeLineEntry(photos:java.util.ArrayList[String],
                          locationName:String,
                          location:BigInteger,
                          start:GregorianCalendar,
                          end:GregorianCalendar,
                          lng:Double,
                          lat:Double) extends Serializable{




    def this(photos:java.util.ArrayList[String], locationName:String, location:BigInteger, start:GregorianCalendar, end:GregorianCalendar){
        this(photos, locationName, location, start, end, 0, 0)
    }

  def this(photos:java.util.ArrayList[String], locationName:String, start:GregorianCalendar, end:GregorianCalendar, lng:Double, lat:Double){
        this(photos, locationName, new BigInteger("0"), start, end, lng, lat)
    }    
    
}